pages/extra/events-and-subscriptions.md

```diff
@@ -143,8 +143,8 @@ Subscribing works via WebSockets only. Request a relevant endpoint from your pro
 First, instantiate a subscriber:
 
 ```
-var ankrSDK = AnkrSDKWrapper.GetSDKInstance("https://...");
-var _eventSubscriber = ankrSDK.CreateSubscriber("wss://...");
+var mirageSDK = MirageSDKWrapper.GetSDKInstance("https://...");
```

Comment:
is this our latest flow? @Antivortex

Reply:
This is one of the up-to-date options. There is a simpler way to get a single event with a special EventAwaiter object (i.e. a single log coming from a single smart contract method call) but this flow is valid too. EventAwaiter just wraps the EventSubscriber for a single event handling.